test-requirements.txt

```diff
@@ -7,7 +7,7 @@ pylint==2.6.0
 pylint-plugin-utils>=0.4
 bandit==1.5.1
 ipython<6.0.0
-isort<=4.0.0
+isort>=4.2.5,<5
```

Comment:
Was there a reason you pinned this to <5.0.0?

According to [the changelog for `isort` v5.0.0](https://github.com/pycqa/isort/blob/master/CHANGELOG.md#500-penny---july-4-2020):

> isort now requires Python 3.6+ to run but continues to support formatting on ALL versions of python including Python 2 code.

Which should be fine for this repo now.

If we do need to keep it pinned to <5.0.0, that would also work fine, but I'd like to start documenting why we pin certain packages to certain versions.

If we keep that pinned to a maximum version, it [potentially complicates our life in the future with pylint](https://app.circleci.com/pipelines/github/StackStorm/st2-rbac-backend/551/workflows/5c2127ec-d717-4ef2-ad1a-e6866e7f102a/jobs/2081):

> ERROR: Cannot install -r /tmp/st2/test-requirements.txt (line 6) and isort<=4.0.0 because these package versions have conflicting dependencies.
> 
> The conflict is caused by:
>     The user requested isort<=4.0.0
>     pylint 2.6.0 depends on isort<6 and >=4.2.5
> 
> To fix this you could try to:
> 1. loosen the range of package versions you've specified
> 2. remove package versions to allow pip attempt to solve the dependency conflict

StackStorm/st2-rbac-backend#46 is being held up by this PR due to this, which is not a big deal, it just forces a resolution order as we slowly crawl towards the next major ST2 release.

Reply:
@blag I copied the version that was in st2-auth-ldap that was needed to get the CI working on st2-auth-ldap. And I believe that requirement was for pylint 2.6.0.
BUT that's a bit odd that the same pylint is now saying < 6 not < 5.
I was also seeing the new pip complain if you have one requirement saying packageA >=4.2.5   and packageB say >=4.25 and < 6. I will see if I can get it to pass with isort > 4.2.5, but I think with the new pip that will complain. But I'll double-check in case it was something else.